src/functions/usernameToColor/__snapshots__/usernameToColor.spec.js.snap

```diff
@@ -0,0 +1,209 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`usernameToColor Akeel Robertson has the proper color 1`] = `
+Color {
+  "b": 164,
```

Comment:
That is hard to read, as normally one would expect r,g,b.

Reply:
Yeah, we can probably just convert it to hex for the test and snapshot

Reply:
Done and updated the json in the first post.